### PR TITLE
Feature/timepout issue

### PR DESCRIPTION
**Pull request by vilasma**
Moved the API calls on secondary thread to keep main thread independent and free for UI operation.
below API's I have moved to secondary thread.
1. getSiteId
2. getGdprStatus
3. getCustomConsents

### CODE REVIEW

File: ConsentViewController/Classes/ConsentViewController.swift [hunk 1]
```diff
@@ -21,26 +21,26 @@ import Reachability
...
+ cbw.consentUUID as Any,
+ cbw.getIABVendorConsents(["VENDOR_ID"]),
+ cbw.getIABPurposeConsents(["PURPOSE_ID"]),
+ cbw.getCustomVendorConsents(),
+ cbw.getCustomPurposeConsents()
+ )
+ }
  view.addSubview(consentViewController.view)
  ```
-*/
```
Comment (andresilveirah) [inline ConsentViewController/Classes/ConsentViewController.swift]: @vilasma can you do me a favour and clean the spacing differences? E.g. reseting the lines that don't have code changes

Comment (vilasma) [reply ConsentViewController/Classes/ConsentViewController.swift]: Okay, I will clean the project also I will remove the methods which are commented by me.

**Comment by vilasma** [comment]
I have removed the commented code please have a look and provide your inputs

File: README.md [hunk 2]
```diff
@@ -17,83 +17,93 @@ import UIKit
...
-            "IABConsent_CMPPresent in storage",
-            UserDefaults.standard.string(forKey: ConsentViewController.IAB_CONSENT_CMP_PRESENT) as Any,
-            "IABConsent_SubjectToGDPR in storage",
-            UserDefaults.standard.string(forKey: ConsentViewController.IAB_CONSENT_SUBJECT_TO_GDPR) as Any
-        )
-    }
+var consentViewController: ConsentViewController!
+
+private func buildConsentViewController(showPM: Bool, addToView parentView: UIView) {
+do {
```
Comment (andresilveirah) [inline README.md]: Hi @vilasma can you make sure that the code is correctly indented in the README? I believe the indentation was lost due to copy and paste.

File: ConsentViewController/Classes/SourcePointClient.swift [hunk 3]
```diff
@@ -32,27 +31,27 @@ typealias TargetingParams = [String:Any]
 
 class SourcePointClient {
     private let client: HttpClient
-
+
```
Comment (andresilveirah) [inline ConsentViewController/Classes/SourcePointClient.swift]: Can you also fix these empty spaces that entered in the diff?

File: Example/ConsentViewController/ViewController.swift [hunk 4]
```diff
@@ -9,26 +9,26 @@ import ConsentViewController
 
 class ViewController: UIViewController {
     var consentViewController: ConsentViewController!
-
+
```
Comment (andresilveirah) [inline Example/ConsentViewController/ViewController.swift]: Same here, spaces that entered the diff.

File: ConsentViewController/Classes/ConsentViewController.swift (hunk 1 above)
Comment (andresilveirah) [reply ConsentViewController/Classes/ConsentViewController.swift]: @vilasma it seems that the spaces issue remained. Could you have another look?

**Pull request closed without merging**